rust/db_handling/src/interface_signer.rs

```diff
@@ -759,27 +842,14 @@ impl SeedDraft {
         }
     }
 
-    /// Make json with seed phrase draft.
-    ///
-    /// Resulting json contains a secret and should be handled as such.
-    pub fn print(&self) -> String {
-        let mut out = String::with_capacity(SAFE_RESERVE); // length set here to avoid reallocation
-        out.push('[');
-        for (i, x) in self.saved.iter().enumerate() {
-            if i > 0 {
-                out.push(',')
-            }
-            out.push_str(&format!("{{\"order\":{},\"content\":\"", i));
-            out.push_str(x.word());
-            out.push_str("\"}");
-        }
-        out.push(']');
-        out
+    pub fn draft(&self) -> Vec<String> {
+        self.saved.iter().map(|w| w.word().to_string()).collect()
```

Comment:
Does this ever reallocate? This should be zeroizeable; `pust_str` was here previously to be certain that secrets do not go into macro. Just double-check that this is clean and preallocate vectors and strings if needed.

Ideally we should also have an `unsafe` direct memory access test for this at some point, even if it is inevitably flaky.